setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I'll see about copying the relevant source into the project instead, but is that really better? The main solution already has 4 git submodules linked into it, is adding a couple nuget dependencies really that much worse.

I can understand the desire to avoid dependencies for simple things that can be easily done without them, but that's really not the case here. All of the other options are quite a bit more work and/or add a lot of unimportant bloat to the project.